src/member/tests.py

```diff
@@ -148,6 +149,12 @@ def test_patch_member(self):
         )
         self.assertEqual(response.status_code, HTTP_403_FORBIDDEN)
 
+    def test_disallows_differently_cased_but_same_username(self):
+        """A differently-cased but otherwise same username should not be allowed registration."""
+
+        self.user.username = self.admin_user.username.upper()
+        self.assertRaises(IntegrityError, self.user.save)
```

Comment:
Similar to teams, this should 400 instead of raising an Exception

Reply:
@david-cooke this is testing the database saving part though, not the endpoint. Or do you mean that the endpoint is missing?

Reply:
Do we handle it on registration?

Reply:
No

Reply:
There isn't really any entry point for error handling there from what I gather, so I would vote that we merge it as-is right now and wait for Sentry to throw up 🛷